library/nxos_zone_zoneset.py

```diff
@@ -462,15 +466,15 @@ def main():
         vsan=dict(required=True, type='int'),
         mode=dict(type='str', choices=['enhanced', 'basic']),
         default_zone=dict(type='str', choices=['permit', 'deny']),
-        smart_zoning=dict(type='bool'),
+        smart_zoning=dict(required=True, type='bool'),
```

Comment:
Again why is smart zoning a must? What about people who dont want to use smart zoning? As i said before i quite dont understand the bug here, so could be please send me (even via email is ok) as to what exactly is the bug with the output of "show running zone" as well as the playbook used to run?